src/http/server.rs

```diff
@@ -49,7 +50,7 @@ pub struct Server {
 	local_addr: SocketAddr,
 	/// Channel to send requests to the background task.
 	to_back: mpsc::UnboundedSender<FrontToBack>,
-	/// List of methods (for RPC queries, subscriptions, and unsubscriptions) that have been
+	/// List of methods (for RPC queries and and notifications) that have been
 	/// registered. Serves no purpose except to check for duplicates.
 	registered_methods: Arc<Mutex<HashSet<String>>>,
```

Comment:
Random thought: maybe this `HashSet` could use `Fnv` hashing? Or do we ever expose this on the wire?

Reply:
Should be fine, it's internal in the server